src/couch/src/couch_httpd.erl

```diff
@@ -44,6 +44,10 @@
 -define(HANDLER_NAME_IN_MODULE_POS, 6).
 -define(MAX_DRAIN_BYTES, 1048576).
 -define(MAX_DRAIN_TIME_MSEC, 1000).
+-define(DEFAULT_SOCKET_OPTIONS, "[{sndbuf, 262144}]").
+-define(DEFAULT_AUTHENTICATION_HANDLERS,
```

Comment:
See another comment about splitting strings into multiple lines.

Reply:
However due to the way `couch_util:parse_term/1` your way also works:
```
Eshell V9.3.3.15  (abort with ^G)
1> couch_util:parse_term("{couch_httpd_auth, cookie_authentication_handler},
1>         {couch_httpd_auth, default_authentication_handler}").
{error,{2,erl_parse,"bad term"}}
```

Reply:
Changed.